apps/mdn/k8s/prod/kuma.yaml.j2

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ KUMA_NAME }}
+spec:
+  type: LoadBalancer
```

Comment:
the versions of K8s that we use have master nodes grouped in with the worker nodes in the ELB: https://github.com/mozmeao/infra/issues/153

We'll have to decide if we want to manually create an ELB, as in https://github.com/mozmeao/infra/tree/master/elbs.